Userland/Libraries/LibC/fenv.h

```diff
@@ -32,6 +34,42 @@ typedef struct fenv_t {
     uint32_t __mxcsr;
 } fenv_t;
 
+#elif defined(__aarch64__)
+
+// TODO: Implement this.
+typedef struct fenv_t {
+} fenv_t;
+
+#elif defined(__riscv) && __riscv_xlen == 64
```

Comment:
In the future it might be nice to defer to arch/fenv.h --> arch/riscv64/fenv.h to get these definitions instead of having ifdefs in the main file here.

Reply:
Why putting this off when I can do it now?